x-pack/plugins/security_solution/public/timelines/containers/index.tsx

```diff
@@ -418,19 +426,18 @@ export const useTimelineEvents = ({
     runtimeMappings,
   ]);
 
-  useEffect(() => {
-    if (
-      id !== TimelineId.active ||
-      timerangeKind === 'absolute' ||
-      !deepEqual(prevTimelineRequest.current, timelineRequest)
-    ) {
-      timelineSearch(timelineRequest);
-    }
-    return () => {
-      searchSubscription$.current.unsubscribe();
-      abortCtrl.current.abort();
-    };
-  }, [id, timelineRequest, timelineSearch, timerangeKind]);
```

Comment:
was `timerangeKind` not used?

Reply:
@michaelolo24 , what do you mean? Sorry I do not completely understand.